File: vol2/ch15/LeetCode60.py
class Solution:
    def getPermutation(self, n: int, k: int) -> str:
        x=[i for i in range(1,n+1)]
        cnt=0
        for e in permutations(x):
            cnt+=1
            if cnt==k:break;
        ret=""
        for d in e:
            ret+=str(d)
        return ret 

File: vol2/ch15/LeetCode60.cpp
class Solution {
public:
    string getPermutation(int n, int k) {
        vector<int> x;
        for(int i=1;i<=n;i++) x.push_back(i);
        int cnt=1;
        while(cnt<k) {
            cnt++;
            next_permutation(x.begin(),x.end());
        }
        string ret="";
        for(int e:x) ret+=to_string(e);
        return ret;
    }
};

